esignet-service/src/test/java/io/mosip/esignet/controllers/KeyBindingControllerTest.java

```diff
@@ -263,12 +263,13 @@ public void bindWallet_withAuthChallengeEmptyFactorAndEmptyChallenge_thenFail()
 				.andExpect(status().isOk())
 				.andExpect(jsonPath("$.errors").isNotEmpty()).andReturn();
 
-		List<String> errorCodes = Arrays.asList(INVALID_AUTH_FACTOR_TYPE, INVALID_CHALLENGE, INVALID_CHALLENGE_FORMAT);
+		List<String> errorCodes = Arrays.asList(INVALID_CHALLENGE_FORMAT,INVALID_AUTH_FACTOR_TYPE, INVALID_CHALLENGE_FORMAT_FOR_AUTH_FACTOR_TYPE,INVALID_CHALLENGE);
```

Comment:
Error list should also contain "iNVALID_CHALLENGE" as challenge is set as empty string @pvsaidurga

Reply:
INVALID_CHALLENGE is present at the end @ase-101